lib/src/lints/flutter/prefer_dedicated_media_query_method.dart

```diff
@@ -54,6 +53,29 @@ class PreferDedicatedMediaQueryMethod extends DartLintRule {
   ) {
     if (!context.pubspec.isFlutterProject) return;
 
+    context.registry.addPrefixedIdentifier((node) {
+      final initializer = node;
```

Comment:
Why declare a variable here?

Reply:
Oh I think that was left by mistake. I forgot to remove that. I will resolve this one.